sdks/python/apache_beam/runners/portability/fn_api_runner_transforms.py

```diff
@@ -591,7 +591,6 @@ def sink_flattens(stages, pipeline_context):
         if pcollections[pcoll_in].coder_id != output_coder_id:
           # Flatten inputs must all be written with the same coder as is
```

Comment:
I think this comment would be better like this:
`Flatten requires that all its inputs use the same coder as its output. Add stages to transcode Flatten inputs that do not use the same coder.`

Reply:
Thanks. Done.